docs/mm_quick_start.md

```diff
@@ -0,0 +1,181 @@
+
+**Important**: All models must be in the same directory
+
+**Important:** The checkpoint directory should be removed between consecutive runs of
+the `model-analyzer profile` command.<br><br>
+
+---
+
+The `--run-config-profile-models-concurrently-enable` option tells MA to load and optimize both models concurrently using the [Quick Search](config_search.md#quick-search-mode) algortihm.
+
+This will profile both models concurrently using the quick search algorithm to find the maximal throughput gain for both models by iterating across instance counts and batch sizes. By default, the algorithm is attempting to find the best balance of gains for each model, not the best combined total throughput.
+
+After the quick search completes, MA will then sweep concurrencies for the top three configurations and then create a summary report and CSV outputs.
+
+---
+
+Here is an example result summary, run on a TITAN RTX GPU:
+
+![Result Summary Top](../examples/mm_result_summary_top.jpg)
+![Result Summary Table](../examples/mm_result_summary_table.jpg)
+
+You will note that the top model configuration has a lower total combined throughput than either the 2nd or 3rd configurations. However, it does a better job at increasing the total percentage throughput gain versus the other configurations.
+
+For example, comparing the top config (42,24) to the third best (37,19), you will see that while the gain in `add_sub` throughput is large (~1600 infer/sec) it is only a 3.5% increase. However, this is offset by a decrease in `resnet50_libtorch` throughput (~75 infer/sec), which represents an 8% decrease in throughput. Therefore, the (42,24) is deemed to be a better configuration by MA.
+
+---
+
+The measured data and summary report will be placed inside the
+`./profile_results` directory. The directory will be structured as follows.
+
+```
+$HOME
+  |--- model_analyzer
+              |--- profile_results
+              .       |--- plots
+              .       |      |--- simple
+              .       |      |      |--- add_sub,resnet50_libtorch
+                      |      |              |--- gpu_mem_v_latency.png
+                      |      |              |--- throughput_v_latency.png
+                      |
+                      |--- results
+                      |       |--- metrics-model-inference.csv
+                      |       |--- metrics-model-gpu.csv
+                      |       |--- metrics-server-only.csv
+                      |
+                      |--- reports
+                              |--- summaries
+                              .        |--- add_sub,resnet50_libtorch
+                              .                |--- result_summary.pdf
+```
+
+## `Optional:` Add constraints
+
+---
+
+In the above results you can see a large (472%) throughput increase, but this is at the expense of a large increase in latency. By creating a YAML config file we can specify per model latency constraints:
```

Comment:
I love this example and flow!

Reply:
Thanks! It took me a while to find a combination of models that would produce this result, but I thought it was important to illustrate this example to the user.